lib/qa/authorities/local/table_based_authority.rb

```diff
@@ -0,0 +1,51 @@
+module Qa::Authorities
+  class Local::TableBasedAuthority < Base
+    def self.check_for_index
+      conn = ActiveRecord::Base.connection
+      if conn.table_exists?('local_authority_entries') && !conn.indexes('local_authority_entries').find { |i| i.name == 'index_local_authority_entries_on_lower_label' }
+        Rails.logger.error "You've installed local authority tables, but you haven't indexed the label.  Rails doesn't support functional indexes in migrations, so you'll have to add this manually:\n" \
+          "CREATE INDEX \"index_qa_local_authority_entries_on_lower_label\" ON \"qa_local_authority_entries\" (local_authority_id, lower(label))\n" \
+          "   OR on Sqlite: \n" \
+          "CREATE INDEX \"index_qa_local_authority_entries_on_lower_label\" ON \"qa_local_authority_entries\" (local_authority_id, label collate nocase)\n" \
+      end
+    end
+
+    attr_reader :subauthority
```

Comment:
This is following the pattern of the whole gem.  `Local` is the authority, and `<your-string-here>` is the sub-authority.